doc/whats_new/v1.0.rst

```diff
@@ -209,6 +209,15 @@ Changelog
   an adequate transformer.
   :pr:`18898` by :user:`Oras Phongpanagnam <panangam>`.
 
+:mod:`sklearn.cross_decomposition`
+..................................
+
+- |Fix| :class:`cross_decomposition._PLS` estimators now properly return a
```

Comment:
`_PLS` is private and users don't know which estimators inherit from it, as it's an implementation detail. So we'll need to list them all :) : PLSRegression, PLSCanonical, CCA